### ISSUE TITLE
Support for Steam's achievements

### ISSUE THREAD

**Issue by Br4ssman** [EasyRPG/Player#1453]
I have plans to launch a commercial game using EasyRPG Player, which supports Steam overlay options, but would it be possible to integrate it with the Steam API to unlock achievements?

A bit example of an idea from EN.i (for RPG_RT.exe 1.12a):
https://twitter.com/elsemieni/status/1048674906098061312

Thanks in advance.

**Comment by rohkea** [comment]
> Also, I heard at IRC that you want to make an GPL exception for allow link Player to steamworks api directly (without shims).

Yes, this has been discussed in https://github.com/EasyRPG/Player/issues/167

**Comment by Br4ssman** [comment]
> Just an opinion: Maybe give Steam support is too specific for a project; 
I think this would be good implement it in the main EasyRPG Player, because today is my case, but in the future (or near future) can be other projects that they want to trust its commercial way with EasyRPG.
